lib/reference/properties.mjs

```diff
@@ -618,3 +618,50 @@ export const longhandTimeProperties = new Set([
 ]);
 
 export const shorthandTimeProperties = new Set(['transition', 'animation']);
+
+export const legacyProperties = new Set([
```

Comment:
[ask] Is there a reliable reference to make this list?

Reply:
Yes, I have added two links in [OP](https://github.com/stylelint/stylelint/pull/7764#issue-2355325301).
Only `-apple-dashboard-region` is included by `known-css-property`.

Reply:
Okay, can you add the links to this variable doc? E.g.

```suggestion
// See ...
export const legacyProperties = new Set([
```

Reply:
The issue will do.